pom.xml

```diff
@@ -71,8 +71,8 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-compiler-plugin</artifactId>
           <configuration>
-            <source>1.7</source>
-            <target>1.7</target>
+            <source>1.8</source>
```

Comment:
probably ok for the server, but the converter components probably don't need language level 8